src/test/cypress/integration/Logout.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { artemis } from '../support/ArtemisTesting';
+import { authTokenKey } from '../support/constants';
+import { Course } from '../../../main/webapp/app/entities/course.model';
+import { ModelingExercise } from '../../../main/webapp/app/entities/modeling-exercise.model';
+
+const courseRequests = artemis.requests.courseManagement;
+const users = artemis.users;
+const courseOverview = artemis.pageobjects.course.overview;
+const modelingEditor = artemis.pageobjects.exercise.modeling.editor;
+
+describe('Logout tests', () => {
+    let course: Course;
+    let modelingExercise: ModelingExercise;
+    const studentOne = users.getStudentOne();
+    const admin = users.getAdmin();
+
+    before('Login as admin and create a course with a modeling exercise', () => {
+        cy.login(admin);
+
+        courseRequests.createCourse(true).then((response) => {
+            course = response.body;
+            courseRequests.createModelingExercise({ course }).then((resp: Cypress.Response<ModelingExercise>) => {
+                modelingExercise = resp.body;
+            });
+        });
+    });
+
+    after(() => {
+        if (!!course) {
```

Comment:
Converting a falsy / truthy object to a boolean is not necessary when evaluating an if-statement, JavaScript will do that automatically. 
```suggestion
        if (course) {
```